### DISCUSSION TITLE
Cannot receive webhook from svix-server with python django

### DISCUSSION THREAD

**Discussion by dotrieuhai2k** [svix/svix-webhooks#1123] [General]
I'm fresher in using webhook with svix-server. After i read a docs of svix, i tried implement it but it did not run as expected :
- Message, Application and Endpoint exists in Svix-server. 
   -  My first endpoint is  internal IP address http://127.0.0.1:8001/webhook/ and i add this subnet in file .env : ```SVIX_WHITELIST_SUBNETS=[127.0.0.1/8]```
   -  The second endpoint is play.svix.com
- Django do not receive any request from svix server. i setup follow this [link](https://www.svix.com/guides/receiving/receive-webhooks-with-python-django/)
- play.svix.com also not receive any request

**Comment by tasn** [comment]
The whitelist configuration that you tried seems wrong (it's overly permissive + it'll block paly, try 0.0.0.0/0 to turn it off). Though we are missing a lot of information in this report. What's the error message you're seeing?
